diff -Naur handbrake-0.10.5/make/include/main.defs handbrake-0.10.5.patch/make/include/main.defs
--- handbrake-0.10.5/make/include/main.defs	2016-02-11 21:14:05.000000000 +0100
+++ handbrake-0.10.5.patch/make/include/main.defs	2016-03-26 16:39:44.217723251 +0100
@@ -39,27 +39,11 @@
 endif
 endif
 
-ifneq (,$(filter $(BUILD.system),darwin cygwin mingw))
-    MODULES += contrib/fontconfig
-    MODULES += contrib/freetype
-    MODULES += contrib/fribidi
-    MODULES += contrib/libxml2
-    MODULES += contrib/libass
-    MODULES += contrib/libogg
-    MODULES += contrib/libvorbis
-    MODULES += contrib/libtheora
-    MODULES += contrib/libsamplerate
-    MODULES += contrib/lame
-    MODULES += contrib/x264
-endif
-
-ifeq (1,$(FEATURE.fdk_aac))
-    MODULES += contrib/fdk-aac
-endif
-
-ifeq (1,$(FEATURE.x265))
-    MODULES += contrib/x265
-endif
+MODULES += contrib/libtheora
+MODULES += contrib/libsamplerate
+MODULES += contrib/x264
+MODULES += contrib/fdk-aac
+MODULES += contrib/x265
 
 MODULES += contrib/ffmpeg
 MODULES += contrib/libvpx
