diff -Naur handbrake-0.10.5/contrib/x265/module.defs handbrake-0.10.5.patch/contrib/x265/module.defs
--- handbrake-0.10.5/contrib/x265/module.defs	2016-02-11 21:14:05.000000000 +0100
+++ handbrake-0.10.5.patch/contrib/x265/module.defs	2016-03-26 16:37:32.929517286 +0100
@@ -9,7 +9,7 @@
 X265.CONFIGURE.deps        =
 X265.CONFIGURE.static      =
 X265.CONFIGURE.shared      = -DENABLE_SHARED=OFF
-X265.CONFIGURE.extra       = -DENABLE_CLI=OFF -DHIGH_BIT_DEPTH=OFF -DENABLE_LIBNUMA=OFF
+X265.CONFIGURE.extra       = -DENABLE_CLI=OFF -DHIGH_BIT_DEPTH=OFF -DENABLE_LIBNUMA=OFF -DENABLE_ASSEMBLY=OFF
 
 ifeq (1,$(BUILD.cross))
     ifeq (mingw,$(BUILD.system))
