From 2187a91cd30aa5d0cd7203526740d7fb144e450e Mon Sep 17 00:00:00 2001
From: Stefan Saraev <stefan@saraev.ca>
Date: Sun, 16 Mar 2014 15:48:28 +0200
Subject: [PATCH] nspr: use nsinstall from nss:host

---
 configure |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/configure b/configure
index 98b5dfe..ee18392 100755
--- a/configure
+++ b/configure
@@ -758,7 +758,7 @@ MOZ_OPTIMIZE=
 OBJDIR='$(OBJDIR_NAME)'
 OBJDIR_NAME=.
 OBJDIR_SUFFIX=OBJ
-NSINSTALL='$(MOD_DEPTH)/config/$(OBJDIR_NAME)/nsinstall'
+NSINSTALL=nsinstall
 NOSUCHFILE=/no-such-file
 LIBNSPR='-L$(dist_libdir) -lnspr$(MOD_MAJOR_VERSION)'
 LIBPLC='-L$(dist_libdir) -lplc$(MOD_MAJOR_VERSION)'
-- 
1.7.2.5

