diff -rupN links-2.8-menu/setup.h links-2.8-menu-patched/setup.h
--- links-2.8-menu/setup.h	2014-09-19 23:49:25.241250256 -0400
+++ links-2.8-menu-patched/setup.h	2014-09-19 23:51:36.445246690 -0400
@@ -139,6 +139,7 @@
 #define G_DIALOG_BUTTON_R		" ]"
 
 #define G_LEFT_ARROW			" \342\206\220 "
+#define G_CIRCLED_TIMES			" \342\223\247 "
 #define MENU_SUBMENU			(!F ? cast_uchar ">" : cast_uchar "\342\206\222")
 
 #define G_SCROLL_BAR_WIDTH		12
diff -rupN links-2.8-menu/view.c links-2.8-menu-patched/view.c
--- links-2.8-menu/view.c	2014-09-19 23:49:55.221249439 -0400
+++ links-2.8-menu-patched/view.c	2014-09-20 18:09:04.691458309 -0400
@@ -3562,6 +3562,10 @@ void send_event(struct session *ses, str
 				go_back(ses, 1);
 				goto x;
 			} else
+			if (F && ev->x > ses->term->dev->size.x2 - 16) {
+				exit_prog(ses->term, (void *)(my_intptr_t)(0), ses);
+				goto x;
+			} else
 #endif
 			{
 				struct window *m;
diff -rupN links-2.8-menu/view_gr.c links-2.8-menu-patched/view_gr.c
--- links-2.8-menu/view_gr.c	2014-09-19 23:50:12.981248957 -0400
+++ links-2.8-menu-patched/view_gr.c	2014-09-20 00:01:37.961230351 -0400
@@ -1341,6 +1341,7 @@ void draw_title(struct f_data_c *f)
 	if (b < z) b = z;
 	drv->fill_area(dev, z, 0, b, G_BFU_FONT_SIZE, bfu_bg_color);
 	g_print_text(drv, dev, b, 0, bfu_style_bw, title, &b);
+	g_print_text(drv, dev, b, 0, bfu_style_bw, cast_uchar G_CIRCLED_TIMES, &b);
 	drv->fill_area(dev, b, 0, dev->size.x2, G_BFU_FONT_SIZE, bfu_bg_color);
 	mem_free(title);
 }
