diff -Naur encfs-1.8.1/encfs/Makefile.am encfs-1.8.1.patch/encfs/Makefile.am
--- encfs-1.8.1/encfs/Makefile.am	2015-03-24 21:45:16.000000000 +0100
+++ encfs-1.8.1.patch/encfs/Makefile.am	2016-03-26 23:27:34.029601532 +0100
@@ -125,10 +125,10 @@
     SSL_Cipher.h \
     StreamNameIO.h
 
+if BUILD_MAN
 man_MANS=encfs.1 encfsctl.1
 EXTRA_DIST = encfs.pod encfsctl.pod encfs.1 encfsctl.1 encfs-man.html
 
-if BUILD_MAN
 SUFFIXES = .1 .pod
 # since we have POD2MAN, we can specify how to rebuild encfs.1 if necessary
 .pod.1:
