diff -Naur libpcap-1.8.1/configure.ac libpcap-1.8.1.patch/configure.ac
--- libpcap-1.8.1/configure.ac	2016-10-26 02:07:59.000000000 +0200
+++ libpcap-1.8.1.patch/configure.ac	2017-01-02 00:23:43.997423834 +0100
@@ -491,7 +491,6 @@
 	if test x$with_libnl != xno ; then
 		have_any_nl="no"
 
-                incdir=-I/usr/include/libnl3
                 libnldir=
                 if test x$withval != x ; then
                   libnldir=-L${withval}/lib/.libs
