diff -Naur hid_mapper-2a80972/MapReader.cpp hid_mapper-2a80972.patch/MapReader.cpp
--- hid_mapper-2a80972/MapReader.cpp	2014-08-11 19:23:36.000000000 +0200
+++ hid_mapper-2a80972.patch/MapReader.cpp	2017-01-09 15:44:14.347001727 +0100
@@ -106,7 +106,11 @@
 			i = 0;
 
 			while(i<KEY_NAME_MAXLENGTH && ptr[i]!='\0' && ptr[i]!='\n' && (!isspace(ptr[i])))
-				key_name[i] = ptr[i++];
+                        {
+                                key_name[i] = ptr[i];
+                                i++;
+                        }
+
 			
 			if(i==KEY_NAME_MAXLENGTH)
 			{
