commit a23dd5e9668585e3ef73cdd5d4e6081950f02212
Author: Stefan Saraev <stefan@saraev.ca>
Date:   Sun Aug 11 23:26:51 2013 +0300

    remove util/mtp-hotplug

diff --git a/Makefile.am b/Makefile.am
index c6d46c2..46d808c 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -11,14 +11,5 @@ EXTRA_DIST=libmtp.pc libmtp.sh COPYING README.windows.txt
 if USE_LINUX
 udevrulesdir=@UDEV@/rules.d
 udevrules_DATA=@UDEV_RULES@
-noinst_DATA=libmtp.usermap libmtp.fdi
 
-libmtp.usermap: util/mtp-hotplug
-	util/mtp-hotplug > libmtp.usermap
-@UDEV_RULES@: util/mtp-hotplug
-	util/mtp-hotplug -u -p"@UDEV@" @UDEV_GROUP@ @UDEV_MODE@ > @UDEV_RULES@
-libmtp.fdi: util/mtp-hotplug
-	util/mtp-hotplug -H > libmtp.fdi
-
-CLEANFILES = libmtp.usermap @UDEV_RULES@ libmtp.fdi
 endif
diff --git a/configure.ac b/configure.ac
index 87d7fcc..497b68a 100644
--- a/configure.ac
+++ b/configure.ac
@@ -29,7 +29,7 @@ AC_SUBST(UDEV)
 # attempt to autoprobe the devices.
 #
 # Later rules can then use the identifiers for its processing.
-UDEV_RULES=69-libmtp.rules
+UDEV_RULES=
 AC_ARG_WITH(udev-rules,
     AC_HELP_STRING([--with-udev-rules=NAME],
     [file name for udev rules [default=69-libmtp.rules]]),
